packages/nimble-components/src/table/specs/table-column-width-hld.md

```diff
@@ -35,7 +38,23 @@ There are some column sizing behaviors that we will ultimately expect to support
 
 ### API
 
-To accomodate the various sizing modes of a column, in addition to the other necessary behaviors, we can add the following properties to `ColumnInternals`:
+`Table`
+
+```ts
+    /**
+     * Allows a user to disable interactive column sizing
+     */
+    @attr({ attribute: 'disable-column-sizing', mode: 'boolean' })
+    public disableColumnSizing = false;
+
+    /**
+     * Allows a user to limit interactive column sizing to the available viewport width.
+     */
+    @attr({ attribute: 'restict-sizing-to-viewport', mode: 'boolean'})
```

Comment:
typo (though I'd still vote for other names)
```suggestion
    @attr({ attribute: 'restrict-sizing-to-viewport', mode: 'boolean'})
```

Reply:
Changing to the single `column-resize` attribute for now.